src/content/docs/vulnerability-management/changing-vulnerability-status.mdx

```diff
@@ -0,0 +1,293 @@
+---
+title: Changing vulnerability status
+metaDescription: Use Vulnerability Management to overcome blindspots and assign remediation to developers as a security team.
+freshnessValidatedDate: never
+---
+
+import vmIgnoreEntityVulnList from 'images/security_screenshot-ignore-entity-vulnlist.webp'
+
+import vmIgnoreEntityDetails from 'images/security_screenshot-ignore-entity-details.webp'
+
+import vmIgnoreEntityForm from 'images/security_screenshot-ignore-entity-form.webp'
+
+import vmIgnoreBulkVulnList from 'images/security_screenshot-ignore-bulk-vulnlist.webp'
+
+import vmIgnoreBulkDetails from 'images/security_screenshot-ignore-bulk-details.webp'
+
+import vmIgnoreBulkForm from 'images/security_screenshot-ignore-bulk-form.webp'
+
+import vmAffectedEntityVulnList from 'images/security_screenshot-affected-entity-vulnlist.webp'
+
+import vmAffectedEntityDetails from 'images/security_screenshot-affected-entity-details.webp'
+
+import vmAffectedEntityFiltered from 'images/security_screenshot-affected-entity-filtered.webp'
+
+import vmAffectedEntityForm from 'images/security_screenshot-affected-entity-form.webp'
+
+import vmAffectedBulkVulnList from 'images/security_screenshot-affected-bulk-vulnlist.webp'
+
+import vmAffectedBulkSelection from 'images/security_screenshot-affected-bulk-selection.webp'
+
+import vmAffectedBulkDetails from 'images/security_screenshot-affected-bulk-details.webp'
+
+import vmAffectedBulkForm from 'images/security_screenshot-affected-bulk-form.webp'
+
+Many organizations have an overwhelming number of vulnerabilities detected within their system. But, not all of these vulnerabilities are high-risk, have remediations, or are run on systems of high operational importance or external exposure. NRVM allows you to ignore vulnerabilities on a per-entity basis, this allows for dashboards to only contain vulnerabilities that match your risk threshold and ensure your teams focus on impactful remediations. 
+
+## Requirements 
+
+- Vulnerability Management requires 'write' permissions for the [vulnerabilities capability](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions/#security) in order for a user to change the status of a vulnerability.
+
+## Ignoring a vulnerability
+
+Not all vulnerabilities detected by Vulnerability Management may require immediate attention. Using Vulnerability Management, you can manually ignore vulnerabilities on a per-entity basis. Once you set the status of an instance to ignored, we’ll exclude this instance from the summary tiles and vulnerability list by default. 
```

Comment:
Why might a developer or security person want to ignore a vulnerability? How can we help them make a risk as sessment? For example, in alerts we have this idea of "alert fatigue" -- getting too many notifications about certain behaviors. Can we provide examples of when a vulnerability could be deprioritized, even if it's more high level and not specific?